sitecore/Dockerfile

```diff
@@ -116,8 +121,11 @@ RUN Copy-Item /Config/*.* c:\inetpub\wwwroot\$Env:SITE_NAME\App_Config\Include
 # enable live-mode
 RUN mv C:\inetpub\wwwroot\sitecore\App_Config\Include\Examples\LiveMode.config.example C:\inetpub\wwwroot\sitecore\App_Config\Include\Examples\LiveMode.config
 
+# Add Host	
+RUN /Scripts/Add-Hosts.ps1
+
 # Start site
 RUN Start-WebAppPool -Name $Env:SITE_NAME; `
     Start-Website -Name $Env:SITE_NAME
 
-ENTRYPOINT C:\ServiceMonitor.exe w3svc
+ENTRYPOINT ["powershell.exe","C:/temp/docker-entrypoint.ps1"]
```

Comment:
Nice addition, but please use docker-entrypoint.ps1 in the docker-compose.yml file. We want the default Docker image to be able to start without requiring a `C:\workspace` directory.